docs/get-started/enterprise/helm-guide.md

```diff
@@ -39,11 +39,12 @@ on a Kubernetes cluster. It requires a valid Feldera Enterprise license
    ```bash
    ACCOUNT_ID="00000000-0000-0000-0000-000000000000"  # Set to own
    LICENSE_KEY="00000000-0000-0000-0000-000000000000"  # Set to own
+   VERSION="0.69.0" # Replace by the desired version.
```

Comment:
@blp should be 0.69.0-amd64 (or arm64) throughout. See https://docs.feldera.com/get-started/enterprise/helm-guide